front/context/src/ApiRxContext.tsx

```diff
@@ -38,14 +38,18 @@ export function ApiRxContextProvider(
     // We want to fetch all the information again each time we reconnect. We
     // might be connecting to a different node, or the node might have changed
     // settings.
-    const subscription = apiRx.isReady.subscribe(() => {
-      l.log(`Api ready.`);
+    let subscription: any;
```

Comment:
This is not nice, neither I nor the quick fix could find the type `Subscription`...